torch/_dynamo/source.py

```diff
@@ -376,32 +368,30 @@ def name(self):
 
 
 @dataclasses.dataclass(frozen=True)
-class SuperSource(Source):
+class SuperSource(ChainedSource):
     type: Source
-    obj: Source
```

Comment:
This BC breaking change feels like a bug farm. I suggest adding `kw_only`

Reply:
fwiw this class is barely used - we could always make another subclass of MultiBaseSource lol

Reply:
Added a comment.